### DISCUSSION TITLE
How to insert inline math in VScode Visual mode?

### DISCUSSION THREAD

**Discussion by DayuanJiang** [quarto-dev/quarto-cli#7232] [Q&A] [answered]
### Description

If I type `$` in visual mode in vscode, the editor will convert it to `\$` automatically.
How can I avoid this?

**Comment by mcanouil** [comment, marked as answer]
If you use the Visual Editor, then use the Visual Editor ;)

<img width="911" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/244d7e34-9f8b-417d-9e03-5101edf6ce80">

Edit: see <https://quarto.org/docs/authoring/markdown-basics.html#equations> for markdown syntax.

**Comment by DayuanJiang** [comment]
I figure out a way to type the inline math. That is to add a space before the equation.
For example, `if$j$` will be convert to  `if\$j\$`, but `if $j$` will not be converted.

**Reply by mcanouil** [reply]
To note, providing an example would have solved this way earlier.
It is always welcome and beneficial to provide a reproducible example when reporting an issue or asking about a behaviour in a tool.

Note that your answer is not the answer to your question it is a fix to a typo in your code, see <https://quarto.org/docs/authoring/markdown-basics.html#equations>.
